haystack/nodes/file_converter/pdf_ocr.py

```diff
@@ -95,7 +105,7 @@ def convert(
 
         pages = []
         try:
-            images = convert_from_path(file_path, first_page=start_page, last_page=end_page)
+            images = convert_from_path(file_path, first_page=start_page, last_page=end_page)  # type: ignore
```

Comment:
Is this `# type: ignore` necessary? I don't get any warning running `mypy` locally. 🤔

Reply:
Yes, because the original package has a bad typing declaration:

`haystack/nodes/file_converter/pdf_ocr.py:98: error: Argument "last_page" to "convert_from_path" has incompatible type "Optional[int]"; expected "int"  [arg-type]`